curriculum/challenges/_meta/accessibility-quiz/meta.json

```diff
@@ -0,0 +1,282 @@
+{
+  "name": "Accessibility Quiz",
+  "isUpcomingChange": true,
+  "dashedName": "accessibility-quiz",
+  "order": 42,
```

Comment:
nice order 🙂

Reply:
something about [42](https://github.com/freeCodeCamp/freeCodeCamp/pull/42876/files#diff-98b7f6910e611f654a026c30e53d5213519e40e823c1d6cf1746d51f0b150ffeR5) that Shaun gravitates towards

Reply:
This is just what the helper script gives by default 🤷‍♂️ . I am convinced it is @ojeytonwilliams playing with _The Hitchhiker's Guide to the Galaxy_